Jenkins_setup.md

```diff
@@ -142,6 +146,52 @@ The teardown script is entered into the Jenkins Post-Build Actions dialog like:
 
 ![Jenkins Build dialog](pictures/jenkins_post_build.png)
 
+## Setting up a bare metal Jenkins Slave Node
+
+### Setting up a jenkins user
+
+Generally, Jenkins Master needs to log into the slave node as `jenkins` user to run the
+all Jenkins job.
+So you need to create a new user `jenkins` on your slave node and add this new user to
+the `sudo` group, which grants it `sudo` priviledges.
+
+```
+$ sudo adduser jenkins
+$ sudo usermod -aG sudo jenkins
+```
+
+Not only just having sudo priviledges, the `jenkins` user also needs to be able to run `sudo`
+without a password.
+Log into your slave node as root and open the `/etc/sudoers` file with `visudo` editor.
+> **Note** You should never edit `/etc/sudoers` with a regular text editor, such as `Vim`
+> or `nano`, because they do not validate the syntax like the `visudo` editor.
+```
+sudo visudo
+```
+Add following line at the end of the `/etc/sudoers` file.
+```
+jenkins  ALL=(ALL) NOPASSWD:ALL
+```
+
+### Providing ssh private keys
+
+We need your host ssh private keys, which will lodge in the Jenkins master private key store.
+The host machine should be the one in which you could remotely ssh into the slave CI Node
+as `jenkins` user.
+Generally, You could find the private ssh keys in `~/.ssh/id_rsa` in your host machine.
+
+> **Note** You could contact Graham Whaley ([`grahamwhaley`](https://github.com/grahamwhaley)) or
+> Salvador Fuentes ([`chavafg`](https://github.com/chavafg)) to upload your ssh private keys.
+
+### Pre-installled requirements
+
+You need to pre-install the following items in your bare metal Jenkins Slave CI node:
+
+- `JRE` for the Jenkins agent
+- `golang`
+   a not-too-old version of go, lately the CI scripts will update it to the correct version.
```

Comment:
A know this is awkward to explain but this is rather vague - how old is "too old"? Any thoughts on this @chavafg, @grahamwhaley?

Reply:
You could reference our [verions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml) file and follow our 'minimum golang version' as recorded in there.